library/portable-simd/crates/core_simd/src/vector.rs

```diff
@@ -1047,6 +1047,7 @@ where
     LaneCount<N>: SupportedLaneCount,
     T: SimdElement,
 {
+    /// Use `from_array` to load the array into a new `Simd`
```

Comment:
I don't think "`From<[T; N]>` uses `from_array`" documentation is very insightful.